docs/source/gbd2019_models/risk_exposures/stunting/index.rst

```diff
@@ -0,0 +1,337 @@
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+
+.. _2019_risk_exposure_child_stunting:
+
+==============
+Child stunting
+==============
+
+.. contents::
+  :local:
+
++-------------------------------------------------+
+| List of abbreviations                           |
++=======+=========================================+
+| TMREL | theoretical minimum risk exposure level |
++-------+-----------------------------------------+
+| CGF   | child growth failure composed of wasting|
+|       | stunting and underweight                |
++-------+-----------------------------------------+
+| HAZ   | Height for age                          |
++-------+-----------------------------------------+
+| DD    | Diarrheal disease                       |
++-------+-----------------------------------------+
+| LRI   | lower respiratory tract infection       |
++-------+-----------------------------------------+
+| MSLS  | measles                                 |
++-------+-----------------------------------------+
+
+
+.. note:: 
+   
+   Should we call this 2020 and have this under 2020 exposure? How do we deal with the different age groups? 
+
+.. _stunting1.0:
+
+Risk Exposure Overview
+++++++++++++++++++++++
+
+Malnutrition is an imbalance between the body’s needs and its use and intake of nutrients. The imbalance can be caused by poor or lacking diet, poor hygiene, disease states, lack of knowledge, and cultural practices, among others. Underweight, stunting, wasting, obesity, and vitamin and mineral deficiencies are all forms of malnutrition. 
+
+**Chronic malnutrition**, also referred to as stunting ...
+
+.. note::
+  Include here a clinical background and overview of the risk exposure you're modeling. Note that this is only for the exposure; you will include information on the relative risk of the relevant outcomes, and the cause models for those outcomes, in a different document.
+
+.. _stunting1.1:
+
+Risk Exposures Description in GBD
++++++++++++++++++++++++++++++++++
+
+.. _stunting1.1.1:
+
+Case definition
+---------------
+
+Stunting, a sub-compoonent indicator of child growth failure (CGF), is based on a categorical definition using the WHO 2006 growth standards for children 0-59 months. Definitions are based on Z-cores from the growth standards, which were derived from an international reference population. Mild, moderate and severe categorical prevalences were estimated for each of the three indicators. Theoretical minimum risk exposure level (TMREL) for stunting was assigned to be greater than or equal to one standard deviation below the mean (-1 SD) of the WHO 2006 standard weight-for-height curve. This has not changed since GBD 2010.
```

Comment:
typo "compoonent"

Reply:
also "Z-cores"

Reply:
^ I think it's worth mentioning which Z-score, here, to differentiate from other CGF categories. (Height for age, i think?)